components/WheelContainer/WheelContainer.js

```diff
@@ -14,67 +13,57 @@ const StyledSlotWrapper = styled.div`
 `;
 
 const StyledSlot = styled.div`
+  font-size: 150%;
   display: flex;
-  height: 200px;
+  height: 240px;
   width: 6rem;
   flex-direction: column;
   align-items: center;
-  gap: 10px;
   border: 1px solid #bababa;
   border-radius: 5px;
   background-color: #f8f8f8;
-  @media (min-width: 400px) {
-    font-size: 150%;
+  @media (min-width: 414px) {
+    font-size: 175%;
     border-radius: 30px;
     background-color: #f8f8f8;
     display: flex;
-    width: 80%;
+    width: 7.5rem;
     height: 280px;
     align-items: center;
-    margin-bottom: 50px;
     border: 1px solid #bababa;
```

Comment:
Safe to remove as well, since `border: 1px solid #bababa` was already set in L23/22